app/com/linkedin/drelephant/ElephantContext.java

```diff
@@ -103,6 +105,7 @@ private void loadConfiguration() {
     loadJobTypes();
 
     loadGeneralConf();
+    loadQuartzConf();
```

Comment:
@amruth
is there a better way to do this? Looks like Quartz is getting tightly coupled with ElephantContext

Reply:
Yes, this should definitely go behind an interface